apps/frontend/src/app/xformation-editor/xformation-editor.component.ts

```diff
@@ -1,21 +1,55 @@
 /* eslint-disable @angular-eslint/component-selector */
 import { Component } from '@angular/core';
 import { FormGroup, FormControl } from '@angular/forms';
+import { FormBuilder } from '@angular/forms';
+import { FormArray } from '@angular/forms';
+
+interface Transform {
+  name: any;
+  description: any;
```

Comment:
Don't use any. Add the proper types here so typescript can help you in the future.